tests/api-tests/fields/filter.test.ts

```diff
@@ -13,10 +13,7 @@ multiAdapterRunners().map(({ runner, provider }) =>
   describe(`${provider} provider`, () => {
     testModules
       .map(require)
-      .filter(
-        ({ skipCrudTest, unSupportedAdapterList = [] }) =>
-          !skipCrudTest && !unSupportedAdapterList.includes(provider)
-      )
+      .filter(({ unSupportedAdapterList = [] }) => !unSupportedAdapterList.includes(provider))
```

Comment:
We'd been doing an unintended filter of the tests to run here, so that's now been removed.